### PR TITLE
Send newest install logs first

### PR DESCRIPTION
**Pull request by zanderz**
@maxtaco @patrickxb

### CODE REVIEW

File: go/client/cmd_log_send_windows.go [hunk 1]
```diff
@@ -58,7 +58,8 @@ func newScannerUTF16or8(filename string) (utfScanner, error) {
 func GetInstallLogPath() (string, error) {
 	// Get the 3 newest keybase logs - sorting by name works because timestamp
 	keybaseLogFiles, err := filepath.Glob(os.ExpandEnv(filepath.Join("${TEMP}", "Keybase*.log")))
-	sort.Strings(keybaseLogFiles)
+	sort.Sort(sort.Reverse(sort.StringSlice(keybaseLogFiles)))
```
Comment (maxtaco) [inline go/client/cmd_log_send_windows.go]: is 3 enough? My failed install had about ~6 from today?

Comment (zanderz) [inline go/client/cmd_log_send_windows.go]: I didn't want to spam the server too badly, should I make it 6? 3 usually covers one normal upgrade.

Comment (maxtaco) [inline go/client/cmd_log_send_windows.go]: 6 is ok

**Comment by maxtaco** [comment]
👍

**Pull request merged**